src/tests/EdFi.DataManagementService.Tests.E2E/StepDefinitions/StepDefinitions.cs

```diff
@@ -66,6 +66,7 @@ public async Task WhenAPUTRequestIsMadeToWith(string url, string body)
         public async Task WhenAGETRequestIsMadeTo(string url)
         {
             url = $"data/{url.Replace("{id}", _id)}";
+            Console.WriteLine("GET URL: " + url);
```

Comment:
Need to remove the `console.WriteLine`

Reply:
Now I see what you were asking about this morning. The way to do this is to inject `TestLogger _logger` into the constructor of this class on line 15. then when you want something logged do `_logger.log.Information(...)`

Reply:
Removed